apps/dataframe/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2016 EURL Tweag & LeapYear Technologies, Inc.
```

Comment:
Actually, in this case, I propose that it be either one or the other. If the code in this changeset was so much based on Tweag code, then let Tweag be the copyright holder. If the code is entirely new, LeapYear is the copyright holder. The code author knows whether the code is entirely new, or not.

I think LeapYear is the correct choice.